tests/ui/test_ui.py

```diff
@@ -68,18 +68,21 @@ def test_set_footer_text_specific_text(self, view, text='blah'):
 
     def test_set_footer_text_with_duration(self, view, mocker,
                                            custom_text="custom", duration=5.3):
-        mocker.patch('zulipterminal.ui.View.get_random_help',
-                     return_value=['some help text'])
-        mock_sleep = mocker.patch('time.sleep')
+        mocker.patch('zulipterminal.ui.View._reset_footer_text')
 
         view.set_footer_text([custom_text], duration)
 
-        view._w.footer.set_text.assert_has_calls([
-            mocker.call([custom_text]),
-            mocker.call(['some help text'])
-        ])
+        view._w.footer.set_text.assert_called_once_with([custom_text])
+        view.controller.update_screen.assert_called_once_with()
+        view._reset_footer_text.assert_called_once_with(duration)
+
+    def _reset_footer_text(self, view, mocker, duration=46.2):
```

Comment:
pytest won't run this unless it starts with `test__reset_footer_.....`. 

Do you want to remove the two blank lines below (81, 83)? It feels pretty a little odd. And if we can save some space and not decrease readability, we should go for it.

Reply:
Oops, good catch - the tests weren't running.

@neiljp Suggests having blank lines surrounding the method under test :sweat_smile:

Reply:
The blank lines clarify the arrange, act and assert segments. Personally, I like having them. :wink:

Reply:
While this does add extra spacing, this is a style I've been moving towards for the reasons that @preetmishra mentioned.

That said, if a test needs zero setup, I might be tempted to avoid the one blank line, but consistency would still be good and we don't currently have many of those - perhaps unfortunately.

Reply:
Sure looks like a cleaner style. 👍